api/src/pcapi/core/finance/api.py

```diff
@@ -1817,17 +1830,15 @@ def find_reimbursement_rule(rule_reference: str | int) -> models.ReimbursementRu
 
 
 def _make_invoice_line(
-    group: models.RuleGroup,
-    pricings: list,
-    line_rate: decimal.Decimal | None = None,
+    group: models.RuleGroup, pricings: list, line_rate: decimal.Decimal | None = None, is_incident_line: bool = False
 ) -> tuple[models.InvoiceLine, int]:
     reimbursed_amount = 0
     flat_lines = list(itertools.chain.from_iterable(pricing.lines for pricing in pricings))
-    # positive
+    # ingoing
     contribution_amount = sum(
         line.amount for line in flat_lines if line.category == models.PricingLineCategory.OFFERER_CONTRIBUTION
     )
-    # negative
+    # outgoing
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
** ❓ question:** est-ce qu'il est indiqué quelque part le signe mathématique de ingoing/outgoing ? ‏

Reply:
Il faudrait en fait trouver le bon wording ici. Car à l'origine, les montant OFFERER_REVENUE par exemple étaient toujours négatifs. Sauf qu'en intégrant les incidents dans la facture, nous avons une valeur positif pour annuler le trop perçu en question. J'avais donc remplacé le siigne mathématique par `outgoing` mais même là ca reste incorrect